src/HttpTracer/HttpMessageParts.cs

```diff
@@ -6,19 +6,15 @@ namespace HttpTracer
     public enum HttpMessageParts
```

Comment:
I have confirmed this change is not breaking
![image](https://user-images.githubusercontent.com/471626/128063831-eef170a9-c877-479b-803a-728cfaf1afc5.png)